iothub_client/samples/iothub_client_c2d_streaming_proxy_sample/iothub_client_c2d_streaming_proxy_sample.c

```diff
@@ -244,15 +244,15 @@ static UWS_CLIENT_HANDLE create_websocket_client(DEVICE_STREAM_C2D_REQUEST* stre
     size_t host_length;
     const char* path;
     size_t path_length;
-    int port;
+    size_t port;
```

Comment:
>size_t port; [](start = 4, length = 12)

Define port as size_t is odd. It should be uint16_t or uint32_t.

Reply:
size_t is uint32_t on 32 bit machines.

Reply:
Sure, but the meaning is different. size_t is for size, this port is probably a number.

---
In reply to: [338800104](https://github.com/Azure/azure-iot-sdk-c/pull/1282#discussion_r338800104) [](ancestors = 338800104)